Modules/CodeTrek/Resources/views/modals/add-applicant-modal.blade.php

```diff
@@ -0,0 +1,88 @@
+@section('popup',' Add new applicant')
+
+<button type="button" class="btn btn-success float-right" data-toggle="modal" data-target="#photoGallery">
+    <a><i class="fa fa-plus"></i>@yield('popup')</a>
+</button>
+<div id="add_applicant_details_form">
+<div class="modal fade" id="photoGallery" tabindex="-1" role="dialog" aria-labelledby="photoGalleryLabel" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
+        <div class="modal-content">
+            <form action="" method="POST" >
+            <div class="modal-header">
+              <h5 class="modal-title" id="exampleModalLabel"><strong>Add new applicant</strong></h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="card-body">
+                <div class="form-row">
+                    <div class="form-group col-md-5">
+                        <label for="first_name" class="field-required">First Name</label>
+                        <input type="text" class="form-control" name="name" id="first_name" placeholder="Enter first name"
```

Comment:
Just check the [naming convention](https://github.com/ColoredCow/portal/blob/master/docs/variable-naming-convention.md) that we use for different attributes.  

```suggestion
                        <input type="text" class="form-control" name="first_name" id="firstName" placeholder="Enter first name"
```

Reply:
Correct the naming convention every where

Reply:
done sir